packages/project/src/shared/styles/css.ts

```diff
@@ -0,0 +1,54 @@
+import { createCssEngine } from "@webstudio-is/css-engine";
+import type { Asset } from "@webstudio-is/asset-uploader";
+import type { Tree } from "@webstudio-is/project-build";
+import {
+  getComponentMeta,
+  getComponentNames,
+  idAttribute,
+} from "@webstudio-is/react-sdk";
+import type { Build } from "../schema";
+import { addGlobalRules } from "./global-rules";
+import { getStyleRules } from "./style-rules";
+
+type Data = {
+  assets: Asset[];
+  build?: Build;
+  tree?: Tree;
+};
+
+export const generateCssText = (data: Data) => {
```

Comment:
So we can import it into SaaS and be happy

Reply:
I thought we already did, this is why we originally created it